pages/about.js

```diff
@@ -1,8 +1,9 @@
 import React from 'react'
 
-export default () => (
+const About = () => (
   <main>
     <h1>About us</h1>
   </main>
 );
 
+export default About;
```

Comment:
btw, is this somehow related to 'prefersh', or just cosmetic?

Reply:
This is related to both react-refresh and prefresh because the function needs a name to be able to determine whether or not it's a component